src/lib/components/Appbar.svelte

```diff
@@ -1,18 +1,25 @@
 <script>
```

Comment:
These items aren't showing on mobile.
![image](https://github.com/Samoa-Stack-Overflow/sso-website/assets/20700150/806e5d03-7bb2-4b7b-9b1b-3d94b24d6ef9)

Reply:
Actually just leave to Footer or just show sidebar.

Reply:
Light/Dark Mode toggle does not work
![image](https://github.com/Samoa-Stack-Overflow/sso-website/assets/20700150/c39d61ba-0709-416e-908e-faf679445079)

Reply:
Done!